src/components/molecules/OfferDetails/OfferDetails.components.tsx

```diff
@@ -71,10 +65,10 @@ export const CompanyLogo: React.FC<ICompanyLogoProps> = forwardRef((props, ref)
 
 export const DetailsList: FC<IDetailsListProps> = ({ items }) => (
   <Box as="ul" sx={{ listStyle: 'none', p: 0, width: '100%', mt: 1 }}>
-    {items.map((item, index) => (
+    {(items || []).map((item, index) => (
```

Comment:
to może lepiej `items?.map` albo jako defaultProps `({ items = [] })` ?

Reply:
nie wiem czym to się różni, ale mogę zmienić